web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -1,12 +1,48 @@
 # PDEP-10: PyArrow as a required dependency for default string inference implementation
 
-- Created: 17 April 2023
-- Status: Accepted
+- Created: 17 April 2023 (updated May 8, 2024)
+- Status: Rejected
 - Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
               [#52509](https://github.com/pandas-dev/pandas/issues/52509)
 - Author: [Matthew Roeschke](https://github.com/mroeschke)
           [Patrick Hoefler](https://github.com/phofl)
-- Revision: 1
+- Revision: 2
+
+# Note
+
+This PDEP was originally accepted on May 8, 2023. However, after reviewing feedback posted
+on the feedback issue [#54466](https://github.com/pandas-dev/pandas/issues/54466), we, the members of
+the core team, have not decided with moving forward with this PDEP for pandas 3.0.
+
+The primary reasons for rejecting this PDEP are twofold:
+
+1) Requiring pyarrow as a dependency causes installation problems.
+   - Pyarrow does not fit or has a hard time fitting in space-constrained environments 
```

Comment:
Within the context of recent conversation I don't think this comment about AWS is true. AWS distributes an official pandas image for lambda which already includes pyarrow, pandas, and NumPy. This is all required by their own "AWS SDK on pandas" library.

The issue more finely scoped I think is that the default wheel installation via pip into a lambda image exceeds the 256 MB limit. Either using the official AWS provided image or using miniconda should not exceed the space limits